packages/test/test-end-to-end-tests/src/test/gc/gcTombstone.spec.ts

```diff
@@ -578,25 +622,23 @@ describeNoCompat("GC tombstone tests", (getTestObjectProvider) => {
                 summarizingContainer,
                 summarizer,
             } = await summarizationWithUnreferencedDataStoreAfterTime(sweepTimeoutMs);
-
-            // Use the request pattern to get the testDataObject - this is unsafe and no one should do this in their
-            // production application - causes a sweep ready loaded error
-            const dataObject = await requestFluidObject<ITestDataObject>(summarizingContainer, unreferencedId);
+            await sendOpToUpdateSummaryTimestampToNow(summarizingContainer);
 
             // The datastore should be tombstoned now
-            await summarize(summarizer);
-
+            const { summaryVersion } = await summarize(summarizer);
+            const container = await loadContainer(summaryVersion);
+            // Requesting the tombstoned data store should succeed since ThrowOnTombstoneUsage is not enabled.
+            // Logs a tombstone and sweep ready error
+            const dataObject = await requestFluidObject<ITestDataObject>(container, unreferencedId);
```

Comment:
You need to explicitly validate that `requestFluidObject` does not throw - that's the test right. With the ThrowOnTombstoneUsage set to false, this wouldn't throw, else it would.